### PR TITLE
Japanese Translation

### PR DESCRIPTION
**Pull request by Phystam**
Japanese Translation

### CODE REVIEW

File: i18n/ja-JP/faq.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Genshin Wishesとは?
+Genshin Wishesはオープンソース(https://github.com/genshin-wishes)の祈願履歴ビューワーです。[Colin Auberger](https://www.linkedin.com/in/colin-auberger/)と[Jimmy Vergerolle](https://vergerolle.fr)等有志によりメンテナンスされています。
```
Comment (Ishou) [inline i18n/ja-JP/faq.md:2]: The brackets [ ] are missing so the link is broken

```suggestion
Genshin Wishesは[オープンソース](https://github.com/genshin-wishes)の祈願履歴ビューワーです。[Colin Auberger](https://www.linkedin.com/in/colin-auberger/)と[Jimmy Vergerolle](https://vergerolle.fr)等有志によりメンテナンスされています。
```

File: i18n/ja-JP/faq.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+4) 青文字で表示された「アカウント管理」をクリックすると、ウェブブラウザが開きます
+5) 表示されたページのURLをコピーし、Genshin Wishesに貼り付けます
+   重要: このURLでパスワードを変更されることはありません。
+
+
+### Consoles
+1) Open your mail in-game during an event
+2) Open the event related mail
+3) There should be a QR code, scan it with your phone
+4) Copy the URL and paste it on Genshin Wishes.
```
Comment (Ishou) [inline i18n/ja-JP/faq.md:41]: This part is not translated

File: i18n/ja-JP/site.json [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+      }
+    },
+    "signIn": "サインイン"
+  },
+  "auth": {
+    "logout.label": "ログアウト"
+  },
+  "banners": {
+    "label": "Banners",
+			"seeWishes": "履歴閲覧",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:76]:
```suggestion
    "seeWishes": "履歴閲覧",
```

File: i18n/ja-JP/site.json [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+      "confirm": {
+        "title": "よろしいですか?",
+        "description": "エクスポートしても祈願履歴はGenshin Wishesにも残ります。"
+      },
+      "action": "祈願履歴をエクスポート!"
+    }
+  },
+  "stats": {
+    "label": "祈願統計",
+			"averageGap":"平均出現頻度",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:181]:
```suggestion
    "averageGap":"平均出現頻度",
```

File: i18n/ja-JP/site.json [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+    "fourStarsList.title": "★4一覧",
+    "bannerActivity.title": "週間祈願統計",
+    "filters": {
+      "filterByEvent": "過去のイベント祈願ごとの絞り込み",
+      "selectEvent": "イベント祈願を選択してください",
+      "allEvents": "全イベント"
+    }
+  },
+  "nav": {
+    "helpUs.label": "Want to help us?",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:198]: Missing translation

File: i18n/ja-JP/site.json [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+    "bannersNav": "祈願一覧",
+    "itemNotFound": "アイテムが見つかりません。データベースを更新するまでお待ち下さい！",
+    "itemType$": {
+      "Character": "キャラクター",
+      "Weapon": "武器"
+    },
+    "filters": {
+      "freeText": "名前で絞り込み",
+      "itemType": "タイプで絞り込み",
+      "rank": "Filter by rank",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:226]: Missing translation

File: i18n/ja-JP/instructions/console.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+1) Open your mail in-game during an event
+2) Open the event related mail
+3) There should be a QR code, scan it with your phone
+4) Copy the URL and paste it on Genshin Wishes.
```
Comment (Ishou) [inline i18n/ja-JP/instructions/console.md:4]: To replace with translation once done in faq.md

File: i18n/ja-JP/items.json [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  "1032": "ベネット",
+  "1027": "凝光",
+  "1025": "行秋",
+  "1024": "北斗",
+  "1020": "レザー",
+  "15405": "弓蔵",
+  "15403": "祭礼の弓",
+  "15401": "西風猟弓",
+  "14403": "祭礼の断片",
+  "14402": "The Widsith",
```
Comment (Ishou) [inline i18n/ja-JP/items.json:23]: Missing translation

File: i18n/ja-JP/site.json [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+    "ok": "OK",
+    "close": "閉じる",
+    "previous": "戻る",
+    "next": "進む",
+    "retry": "再読み込み",
+    "soon": "Soon",
+    "beta": "β",
+    "save": "保存",
+    "goBack": "戻る",
+    "moreOf": "More {{ content }}",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:16]: Missing translation

File: i18n/ja-JP/site.json [hunk 10]
```diff
@@ -0,0 +1,305 @@
+{
+  "generics": {
+    "share": "共有",
+    "copy": "コピー",
+    "filter": "絞り込み",
+    "cancel": "キャンセル",
+    "ok": "OK",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:7]: Missing translation? Or is "OK" ok in Japanese? 😄

File: i18n/ja-JP/site.json [hunk 11]
```diff
@@ -0,0 +1,305 @@
...
+    "close": "閉じる",
+    "previous": "戻る",
+    "next": "進む",
+    "retry": "再読み込み",
+    "soon": "Soon",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:12]: Missing translation or is it ok like this?

File: i18n/ja-JP/site.json [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+        "title": "初心者応援祈願",
+        "titleWithCount": "初心者応援祈願 ({{ wishes }}連)"
+      },
+      "PERMANENT": {
+        "title": "通常祈願 - 奔走世間",
+        "titleWithCount": "通常祈願 - 奔走世間 ({{ wishes }}連目)"
+      },
+      "CHARACTER_EVENT": {
+        "title": "Character banner",
+        "titleWithCount": "Character banner ({{ wishes }} wishes)"
```
Comment (Ishou) [inline i18n/ja-JP/site.json:293]: Missing translation

File: i18n/ja-JP/site.json [hunk 13]
```diff
@@ -0,0 +1,305 @@
...
+      },
+      "WEAPON_EVENT": {
+        "title": "イベント祈願 - 神鋳賦形",
+        "titleWithCount": "イベント祈願 - 神鋳賦形 ({{ wishes }}連目)"
+      }
+    }
+  },
+  "footer": {
+    "disclaimer": "Genshin Wishesは有志の非公式サイトです。原神およびmiHoYoはmiHoYoの商標です。Genshin Impact c miHoYo.",
```
Comment (Ishou) [inline i18n/ja-JP/site.json:302]: Genshin Impact => 原神

**Review by Ishou — CHANGES_REQUESTED** [review]
Could you review the changes お願いします 🙏 😄

File: i18n/ja-JP/site.json (hunk 11 above)
Comment (Phystam) [reply i18n/ja-JP/site.json:12]: "Soon" is commonly used in Japanese!

File: i18n/ja-JP/site.json (hunk 10 above)
Comment (Phystam) [reply i18n/ja-JP/site.json:7]: OK is OK!

Comment (Phystam) [reply i18n/ja-JP/site.json:7]: or "適用" is another choice for the translation

**Review by Ishou — APPROVED** [review]

**Pull request merged**